README.md

```diff
@@ -28,11 +28,31 @@ For installation & getting started instruction please refer to [our docs](https:
 
 ## Additional Information
 
-Please visit our [PSoC products page](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) for datasheets, pin out diagrams, and additional technical information.
+Please visit our [PSOC products page](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) for datasheets, pin out diagrams, and additional technical information.
 
 ## Contributing
 
-We welcome community contributions! Please refer to our [contribution guidelines](CONTRIBUTING.md). Your help is greatly appreciated 👐
+We welcome community contributions! Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated** 👐.
+
+If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
```

Comment:
Very nice text :) 
Shouldn´t we ask contributors to ask the feature with code owners if the change is a "big"?